features/step_definitions/data.js

```diff
@@ -48,41 +48,26 @@ module.exports = function () {
         q.awaitAll(callback);
     });
 
-    this.Given(/^the node map$/, (table, callback) => {
-        let q = d3.queue();
-
-        let addNode = (name, ri, ci, cb) => {
-            if (name) {
-                let nodeWithID = name.match(/([a-z])\:([0-9]*)/);
-                if (nodeWithID) {
-                    let nodeName = nodeWithID[1],
-                        nodeID = nodeWithID[2];
-                    if (this.nameNodeHash[nodeName]) throw new Error(util.format('*** duplicate node %s', name));
-                    let lonLat = this.tableCoordToLonLat(ci, ri);
-                    this.addOSMNode(nodeName, lonLat[0], lonLat[1], nodeID);
-                } else {
-                    if (name.length !== 1) throw new Error(util.format('*** node invalid name %s, must be single characters', name));
-                    if (!name.match(/[a-z0-9]/)) throw new Error(util.format('*** invalid node name %s, must me alphanumeric', name));
-
-                    if (name.match(/[a-z]/)) {
-                        if (this.nameNodeHash[name]) throw new Error(util.format('*** duplicate node %s', name));
-                        let lonLat = this.tableCoordToLonLat(ci, ri);
-                        this.addOSMNode(name, lonLat[0], lonLat[1], null);
-                    } else {
-                        if (this.locationHash[name]) throw new Error(util.format('*** duplicate node %s'), name);
-                        let lonLat = this.tableCoordToLonLat(ci, ri);
-                        this.addLocation(name, lonLat[0], lonLat[1], null);
-                    }
-                }
-
-                cb();
+    this.Given(/^the node map$/, (docstring, callback) => {
+        var q = d3.queue();
+
+        var addNode = (name, ri, ci, cb) => {
+            var lonLat = this.tableCoordToLonLat(ci, ri);
+            if (name.match(/[a-z]/) ) {
+                if (this.nameNodeHash[name]) throw new Error(util.format('*** duplicate node %s', name));
+                this.addOSMNode(name, lonLat[0], lonLat[1], null);
```

Comment:
Is this removing the `node:id` feature here, or is this the combination with https://github.com/Project-OSRM/osrm-backend/pull/2979/files#diff-c41ecde0b11b4c1dc39c3b4a6f814478R109? Having IDs is kind of important, potentially. But I don't see that we need to do it in the docstring. 
If this is what happens in added table, I would be totally fine with it. No need to have it in the docstring.